Dockerfile

```diff
@@ -27,10 +28,12 @@ RUN apt-get update \
 RUN localedef -i en_US -f UTF-8 en_US.UTF-8 \
 	&& useradd -m -s /bin/bash linuxbrew \
 	&& echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
-ADD . /home/linuxbrew/.linuxbrew/Homebrew
+COPY . /home/linuxbrew/.linuxbrew/Homebrew
 ARG FORCE_REBUILD
+
+# hadolint ignore=DL3003
 RUN cd /home/linuxbrew/.linuxbrew \
-	&& mkdir -p bin etc include lib opt sbin share var/homebrew/linked Cellar \
+  && mkdir -p bin etc include lib opt sbin share var/homebrew/linked Cellar \
```

Comment:
It looks like the other lines use a tab to indent and this changes it to spaces on this line. I'm fine with either tabs or spaces, as long as its consistent throughout the file.

Reply:
I made them consistently spaces, as this is most consistent with the rest of this repo and people's editor config.

Reply:
TIL about Vim's `:retab` command!